pkg/codec/vpx/vpx_test.go

```diff
@@ -87,3 +89,72 @@ func TestImageSizeChange(t *testing.T) {
 		})
 	}
 }
+
+func TestRequestKeyFrame(t *testing.T) {
+	for name, factory := range map[string]func() (codec.VideoEncoderBuilder, error){
+		"VP8": func() (codec.VideoEncoderBuilder, error) {
+			p, err := NewVP8Params()
+			return &p, err
+		},
+		"VP9": func() (codec.VideoEncoderBuilder, error) {
+			p, err := NewVP9Params()
+			// Disable latency to ease test and begin to receive packets for each input frame
+			p.LagInFrames = 0
```

Comment:
From my understanding, LagInFrames defaults to 0, similar to VP8. Why VP9 need to be configured differently?

Reply:
It seems that 0 is not the default for VP9 (I don't remember where I found this, I will update if I find it). This leaded in test to get empty packets.
For the first tests, it was not a big deal because we didn't care about the content of the packets.

For me, I need the packets to be sent directly for being able to check if I really get a key frame when requested.